whitepaper/chapter01.md

```diff
@@ -0,0 +1,150 @@
+
+The orchestration layer is tasked with composing, decomposing, and reshaping or resizing the resources assigned to specific applications provided by the infrastructure. It does this based on assessments and monitoring of performance and availability of the actual infrastructure used by application. In that manner, the applications are the “brains”, directing the orchestration layer towards functional goals and outcomes (for example, low latency or specific uptime and resiliency metrics). Thus, the objective of orchestration is to create an entity that aggregates the microservices into services which can be easily mapped to network functions.
+
+<a name="1.3.3"></a>
+### 1.3.3 Infrastructure, Deployment and Configuration of Cloud Native Systems
+
+[Immutable infrastructure](https://docs.google.com/document/d/1fwV3vZB5IgHb6uUgQpHts9mFR0HheYbIrlg9hGfWYfA) (the orchestrator and all of the software and hardware that it depends on) is provisioned using baked and versioned templates (e.g. server images) or a combination of templates and bootstrapping (some repeatable and versioned process that is applied to the template e.g. Kubeadm).  The underlying infrastructure is not changed after it is made ready for use. New changes to the infrastructure are rolled out as new instances of infrastructure. By separating the application layer from the infrastructure layer, it is possible to iterate faster with minimal dependency risk. Similarly it enables the vision of programmable and software defined infrastructure that can made possible to automate end to end deployment .
+
+In cloud native systems, the atomic unit of deployment is the container, on an immutable and decoupled compute environment that nearly completely abstracts function from the underlying physical infrastructure and network architecture. At present, the most commonly used container environment is an OCI-compliant implementation. A container is different than a Virtual Machine (VM), which was the initial base unit of cloud computing. A VM hosts a Guest OS capable of delivering a service's required compute and networking. This guest runs on top of a hypervisor consuming the virtual infrastructure and orchestration it provides. In contrast, containers run on top of a shared Host OS (with a shared kernel) kernel. This distinction is key because containerisation enables immutability, disposability and resource efficiency in ways that are not easily achievable with VMs.
+
+For Kubernetes specifically, the atomic unit of management is a Pod. A Pod is a set of containers co-located in the same or different host and sharing the same Linux networking namespace.  For telco (and other) cloud native apps, the Pod is the basic building block of deployment, scaling operations, and upgrades.
```

Comment:
This should be updated to be aligned with K8s docs https://kubernetes.io/docs/concepts/workloads/pods/pod/